BPMRFFEApp/Db/bpmrffe.db

```diff
@@ -275,7 +275,16 @@ record(ai, "$(P)$(R)RFFEHeaterBD-RB") {
     field(EGU, "V")
 }
 
-# System reset
+# System operations
+
+record(waveform, "$(P)$(R)RFFEVersion-Cmd") {
+    field(DESC, "Get apps, nuttx, RFFE Git hash & version")
```

Comment:
I'd prefer something shorter. "Get firmware version" is enough, IMO.

How did you get the 140 characters length? Would it be better to support a bit more in case there *is* more?

Reply:
Here's the rationale:
- `3 * 40`: three Git commit hashes, each 40 characters (this is fixed, since it is the `sha1sum` output)
- `+ 3`: commas for separating hashes and version tag
- `+ 3`: three digits for version tag `x.y.z` (yes, they can be more than one digit)
- `+ 2`: two dots from the above number separation
- `+ 4`: `-rcX` release candidate token (single digit)
- `+ 8`: padding for more digits in any of the previous numbers. I chose it so that the resulting number is the next multiple of 10.

Any suggestion on this? I considered that the version is following only this pattern of semantic versioning with release candidate tokens. This is true for all existing version tags.

I shall use this more detailed explanation in the commit message. I see I missed at least a comma in the message.